articles/api-auth/token-introspection/app-specific-passwords.md

```diff
@@ -0,0 +1,84 @@
+---
+description: How to use app-specific password flows
+---
+
+# Application-Specific Passwords
+
+An **application-specific password** (ASP) can be used to get you limited access to a protected resource, such as an API. In Auth0, you can use them in one of two ways:
+
+* As a replacement for the password portion of your credentials when using basic authentication. More specifically, instead of providing your username/password combination, you'd provide your username/ASP combination;
+* As an alternative to Bearer tokens when calling an Auth0 API endpoint.
+
+## Prerequisites
+
+Before you can use Application-Specific Passwords (ASP), you'll need to complete the following steps:
+
+1. Provide your API's public key to Auth0
+2. Set up the client grant for ASP
+
+### Step 1: Provide Your API's Public Key to Auth0
+
+There are three ways by which Auth0 can obtain the public key:
+
+1. You can define the `verificationKey` on the API's Auth0 record. This is a JSON Web Key (JWK) or PEM-encoded certificate.
+2. You can define the `verificationLocation` on the API's Auth0 record. This is a URI from which Auth0 fetches JSON Web Keys (JWK).
+3. If the API's `identifier` is `http:`- or `https:`-based (such as `https://example.com/foo`), then Auth0 attempts to fetch the key from the well-known JWK registry of the host (for the provided URI example, Auth0 attempts to fetch from `https://example.com/.well-known/jwks.json`).
+
+### Step 2: Set Up the Client Grant for App-Specific Passwords
+
+Ensure that you've granted the appropriate [scopes](/scopes/current#api-scopes) (such as `read`, `create`, `delete:user_application_passwords`) to the [Client](/clients) you'll be using with the Management API. These will the be the scopes the API can include with newly-issued ASPs.
```

Comment:
> These will the be the scopes the API can include with newly-issued ASPs.
These are the scopes needed for the client to be able to call the Management API endpoints to manage ASPs.